src/OpenTelemetry.Contrib.Extensions.AWSXRay/OpenTelemetry.Contrib.Extensions.AWSXRay.csproj

```diff
@@ -1,13 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net452;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
```

Comment:
If we decide to change minimal .NET Framermwprk it should be raised to .NET Framework 4.6.2. Any older version is not supported by MS. 

This change should be included in CHANGELOG.

Reply:
I am not convinced we need to update, this change is likely something we can revert. The reason why the PR is in draft, besides the missing tests, is that I am struggling big times with my dev setup atm, and this change is because of it ;-)

Reply:
Actually, it's probably better to raise to `net462`. Done in f90fce5.